mmdet/evaluation/metrics/coco_metric.py

```diff
@@ -73,7 +72,7 @@ def __init__(self,
                  metric_items: Optional[Sequence[str]] = None,
                  format_only: bool = False,
                  outfile_prefix: Optional[str] = None,
-                 file_client_args: dict = dict(backend='disk'),
+                 backend_args: dict = None,
```

Comment:
we can also add into config, to give user examples

Reply:
you mean add this in config? but in original config, it doesn't seem to show up. original config also uses default value for file_client_args